src/components/modals/AddCustomLinkModal.test.tsx

```diff
@@ -2,82 +2,49 @@
  * @jest-environment jsdom
  */
 
-import { screen } from '@testing-library/react'
+import { screen, RenderResult } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import React, { createRef } from 'react'
 import { ModalRef } from '@trussworks/react-uswds'
 import { renderWithModalRoot } from '../../testHelpers'
-
+import { axe } from 'jest-axe'
 import AddCustomLinkModal from './AddCustomLinkModal'
 
 describe('AddCustomLinkModal', () => {
-  const mockOnSave = jest.fn()
-  const mockOnCancel = jest.fn()
+  const testHandlers = {
+    onSave: jest.fn(),
+    onCancel: jest.fn(),
+  }
 
-  beforeEach(() => {
+  let html: RenderResult
+  beforeEach(async () => {
     const modalRef = createRef<ModalRef>()
-    renderWithModalRoot(
+
+    html = renderWithModalRoot(
       <AddCustomLinkModal
         modalRef={modalRef}
-        onSave={mockOnSave}
-        onCancel={mockOnCancel}
+        onSave={testHandlers.onSave}
+        onCancel={testHandlers.onCancel}
       />
     )
+    await screen.findByRole('dialog', { name: 'Add a custom link' })
   })
 
-  it('renders a form to add a custom link', () => {
+  it('renders the AddCustomLinkModal component', async () => {
     expect(screen.getByRole('heading')).toHaveTextContent('Add a custom link')
 
-    const labelInput = screen.getByLabelText('Name')
```

Comment:
Most of the testing in this file and `EditCustomLinkModal` were really testing the form. Now that the form is its own component, these tests were refactored and moved to `CustomBookmarkForm.test.tsx`